yaml_armhf/gateway.yml

```diff
@@ -34,8 +34,8 @@ spec:
             memory: 100Mi
           limits:
             memory: 100Mi
-      - name: faas-netesd
-        image: functions/faas-netesd:0.5.2-armhf
+      - name: faas-netes
+        image: functions/faas-netes:0.5.2-armhf
```

Comment:
@mrtind this has to remain as faas-netesd because I haven't pushed any images for armhf or arm64.

Reply:
ok I will revert that.